### ISSUE TITLE
have pipes accept null inputs?

### ISSUE THREAD

**Issue by PlaidPhantom** [dstelljes/luxon-angular#4]
Right now I have a number of optional/nullable DateTime fields in my models, and when I want to format them I have to write some awkward boilerplate to get around it:

```javascript
myDateTime ? (myDateTime | dateTimeToFormat:'MM/DD/YYYY') : null
```

It would be very helpful if the formatting pipes would "fail gracefully" instead of throwing exceptions on null inputs. Perhaps something like:

```javascript
if(!DateTime.isDateTime(value)) // or maybe "value === null || value === undefined"?
    return null;
else
    // do thing
```

If it helps, I'd be happy to make a PR for this functionality.

**Comment by dstelljes** [comment]
We'd probably want to take an approach similar to the [built-in date pipe](https://github.com/angular/angular/blob/8.0.0/packages/common/src/pipes/date_pipe.ts#L172): return null for empty values; throw if the type is wrong. I'll try to get that change made sometime this weekend.

**Comment by dstelljes** [comment]
Added with 37982d215548ecf23fcab7daa7750a4f3c005637, released with 2.0.0.

**Comment by PlaidPhantom** [comment]
Thanks for changing this so quickly!
